schema/src/main/java/org/hl7/v3/RCMRMT030101UK04PlanStatement.java

```diff
@@ -79,7 +79,7 @@
     protected II id;
```

Comment:
Could you mention this class in the schema module's readme file please? (inside 'Manually modified classes' section)